3-Solving-Problems-By-Searching/helpers.js

```diff
@@ -14,48 +12,92 @@ function frontier(){
   return frontier();
 }
 // Default Graph for visualizations
-function makeDefaultGraph(){
+function makeDefaultGraph() {
   defaultGraph = {}
-  defaultGraph.nodes = [
```

Comment:
Yeah. We might try something else later. Maybe add the beautifier as a part of gulp-copyback script?